travis_setup.sh

```diff
@@ -33,10 +33,11 @@ if [ ! -d ~/.modules/mmeds-stable ]; then
     echo "Create mmeds-stable environment";
     conda create --file spec-file.txt -p ~/.modules/mmeds-stable --yes --quiet --copy &>/dev/null;
     ln -sf ~/.modules/mmeds-stable ~/miniconda2/envs/mmeds-stable;
-    source activate mmeds-stable;
-    echo "Install R libraries";
-    Rscript setup.R &> /dev/null;
-    source deactivate;
+    # source activate mmeds-stable;
```

Comment:
Commented out code is no bueno

Reply:
Shhh this isn't even really a part of the project